src/object/obj_rpc.h

```diff
@@ -158,6 +158,7 @@ struct obj_iod_array {
 	((uint32_t)		(orw_start_shard)	CRT_VAR) \
 	((uint32_t)		(orw_flags)		CRT_VAR) \
 	((daos_key_t)		(orw_dkey)		CRT_VAR) \
+	((struct dcs_csum_info)	(orw_dkey_csum)		CRT_PTR) \
 	((struct obj_iod_array)	(orw_iod_array)		CRT_VAR) \
 	((struct dcs_iod_csums)	(orw_iod_csums)		CRT_ARRAY) \
```

Comment:
BTW, a minor issue that orw_iod_csums in input struct and orw_iod_csum (no 's') in output struct. is it intended?
if no I may rename them to be same in my patch.

Reply:
No, they should be the same. My intention is if there are multiple iods, then the csum field/param name should be iods_csums. If just one iod, then the csum field/param should be iod_csums. IODs can have multiple csums.